### PR TITLE
ParentOutdatedAutoMaterializeCondition gql

### PR DESCRIPTION
**Pull request by johannkm**
Add field to gql for which assets we're waiting on

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #14837** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/14837" target="_blank">Graphite</a> 
    * **PR #14838** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/14838" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/14838?utm_source=stack-comment).

File: python_modules/dagster-graphql/dagster_graphql/schema/auto_materialize_asset_evaluations.py [hunk 1]
```diff
@@ -142,7 +149,11 @@ def create_graphene_auto_materialize_condition(
         )
     elif isinstance(condition, ParentOutdatedAutoMaterializeCondition):
         return GrapheneParentOutdatedAutoMaterializeCondition(
-            decisionType=condition.decision_type, partitionKeysOrError=partition_keys_or_error
+            decisionType=condition.decision_type,
+            partitionKeysOrError=partition_keys_or_error,
+            parentAssetKey=condition.parent_asset_key,
+            differentPartitions=condition.different_partitions,
+            differntRepositories=condition.different_repositories,
```
Comment (OwenKephart) [inline python_modules/dagster-graphql/dagster_graphql/schema/auto_materialize_asset_evaluations.py:156]: differnt

**Review by OwenKephart — COMMENTED** [review]
Not sure if this PR or the other one is better for this discussion, but how did we arrive at this set of information to display to the user? In particular, do we need differentPartitions and differentRepositories?

**Comment by johannkm** [comment]
Maybe not, it seemed like we easily had that information so might as well surface more details about why we're waiting

**Comment by OwenKephart** [comment]
@johannkm my feeling was that it might make sense to just link out to the asset definition (from the key), and then users can discover that info for themselves.

not durable if the partitions def or repository changes, but I think that's not a huge deal

**Comment by johannkm** [comment]
I think there's value in being explicit but I don't feel strongly. It's kind of a confusing thing to explain either way ("waiting on upstream instead of materializing in same run as upstream because it has a different partitions def than at least one asset that was selected to materialize with the upstream")

**Comment by OwenKephart** [comment]
@johannkm yeah definitely reasonable -- I guess the way that I look at it, it's actually the _other_ behavior (grouping downstream assets into the same run) that's doing the "special" thing, and that's where "ParentUpdatedMaterializationCondition" doesn't quite make sense (the downstream asset's parent hasn't been updated yet, it just will be by the time that it's executed in the run we kicked off).

So I think it still feels fairly natural to just tell them "we're waiting for X asset to update"

**Comment by johannkm** [comment]
Removed the additional details about why we're waiting, only added the set of asset keys

**Review by sryza — APPROVED** [review]
~~This looks good. Merging. Thanks @renzhe-brian!~~

Oops, wrong PR.

**Comment by sryza** [comment]
However, this PR also LGTM @johannkm

**Pull request merged**